referencia/posintegrado/README.md

```diff
@@ -320,6 +320,44 @@ DATO                    | LARGO     | COMENTARIO
 `<ETX>`                 |  1        | Indica el fin de texto o comando <br><i>valor hexadecimal</i>: `0x03`
 `LRC`                 |  1        | Resultado del calculo del `LRC` del mensaje
 
+### Mensaje de Totales
+
+Esta operación le permitirá a la caja tener un resumen de totales con las transacciones que fueron procesadas.
```

Comment:
You're missing the explanation from `Documentación` alongside the diagrama and the explanation.

Reply:
> You're missing the explanation from `Documentación` alongside the diagrama and the explanation.

There isnt diagram in oficial docs, we create it? :/